packages/opentelemetry-exporter-collector/src/platform/node/utilWithJson.ts

```diff
@@ -55,17 +105,17 @@ export function sendSpansUsingJson(
     headers: {
       'Content-Length': Buffer.byteLength(body),
       'Content-Type': 'application/json',
-      ...collector.headers,
+      ...collectorHeaders,
     },
   };
 
   const request = parsedUrl.protocol === 'http:' ? http.request : https.request;
   const req = request(options, (res: http.IncomingMessage) => {
     if (res.statusCode && res.statusCode < 299) {
-      collector.logger.debug(`statusCode: ${res.statusCode}`);
+      logger.debug(`statusCode: ${res.statusCode}`);
```

Comment:
Nit, optional: consider making `statusCode: ${res.statusCode}` a variable.

Reply:
why?

Reply:
Ah, good question - I should have elaborated.

I had seen it was used both on lines 115 and 118, factoring it out into some variable called status message makes sense to me as a way to convey intent to future developers that the message is supposed to be the same, should they ever change it.

As you've pointed out elsewhere, it's unnecessarily expensive, particularly in this case where it's so simple, but again I just meant it as a conveyance of intent.